### DISCUSSION TITLE
Button in Home Assistant for “Boost charging”

### DISCUSSION THREAD

**Discussion by Koning-Martin** [leeyuentuen/alfen_wallbox#32] [Q&A] [answered]
The Boost option shows as a “**select**.laadpaal” entity in home assistant. I would like to make a big button on my dashboard that I (but more the wife) can push (like a light switch) to turn boost charging “on” and “off”.

I can’t get it to work though. I can successfully create the button, but when I push it I get an error “connection lost”. 

I think I run into this problem because I use a “select” entity with a “switch” button card. B it I can’t find any information about “select” entities or how to handle them right in Home Assistant. 

 Is there an way to accomplish that big button card with the Boost entity? Thank you!

**Comment by leeyuentuen** [comment]
i think i can add extra code to put them in Binary instead of switch. with binary you can easy add them with a button

**Reply by Koning-Martin** [reply]
HA sees it now as a “select.****” entity, so if changing it to binary would fix the (small) issue then that would be great! I think that changing it to a binary sensor makes it also easier to use in automation script, but not sure about that one. I’m not home right now, so can’t check. :-)

**Comment by leeyuentuen** [comment]
i think it should be logic to change then to binary, because it's only on off. select is more for a select in a list of different value

i'll make a change later today.

**Reply by Koning-Martin** [reply]
Thank you! I will try it as soon as it is released! 

I have really no idea how people can actually use that boost function without your (or an fork) integration in Home Assistant. The native MyEve is a pain, takes a while to load and is not user friendly.

Thanks again for your very well working integration!

**Reply by leeyuentuen** [reply, marked as answer]
can you take a look at this beta version: https://github.com/leeyuentuen/alfen_wallbox/releases/tag/2.4.0

**Reply by Koning-Martin** [reply]
Do I have to install it manually, because it is a dev version? I can’t see it in Home Assistant. Noop over here 😅

**Reply by leeyuentuen** [reply]
Go to HACS -> integrations-> search for alfen -> 3 dots on the right -> select redownload -> Select the slider for beta ->  and choose the 2.4.0

**Reply by Koning-Martin** [reply]
Your update (2.4.0) fixed it! It works like a charm, thank you! 
![IMG_1273](https://github.com/leeyuentuen/alfen_wallbox/assets/81510382/03b928d5-6c03-463e-9ae3-486528496249)

**Reply by leeyuentuen** [reply]
great, i'll make that as tested. and will be on the next release
